routesrv/testdata/lb-target-multi.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  namespace: namespace1
+  name: ingress1
+spec:
+  backend:
+    serviceName: service1
+    servicePort: port1
```

Comment:
maybe clarify in the file name that we test `default route`, which is what `spec.backend` defines AFAIR.